### PR TITLE
Italian Fiscal Code regex

### PR DESCRIPTION
**Pull request by mavnt**
**⚠ Pull Requests not made with this template will be automatically closed 🔥**

## Prerequisites
- [x] Have you read the documentation on contributing? https://github.com/bee-san/pyWhat/wiki/Adding-your-own-Regex

## Why do we need this pull request?
Regex for Italian fiscal code (https://en.wikipedia.org/wiki/Italian_fiscal_code).

## What [GitHub issues](https://github.com/bee-san/pyWhat/issues) does this fix?
None.

## Copy / paste of output

Please copy and paste the output of PyWhat with your new addition using an example that tests this addition below:

```console
$ pywhat "RSSMRA00A01H501C"
Matched on: RSSMRA00A01H501C
Name: Italian Fiscal Code
Description: https://en.wikipedia.org/wiki/Italian_fiscal_code
```

### CODE REVIEW

**Review by bee-san — APPROVED** [review]

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -1363,6 +1363,23 @@
          "Invalid": []
       }
    },
+   {
+      "Name": "Italian Fiscal Code",
+      "Regex": "^([A-Za-z]{6}[0-9]{2}[ABCDEHLMPSTabcdehlmpst]{1}[0-9]{2}[A-Za-z]{1}[0-9]{3}[A-Za-z]{1})$",
+      "plural_name": false,
+      "Description": "https://en.wikipedia.org/wiki/Italian_fiscal_code",
```
Comment (piatrashkakanstantinass) [inline pywhat/Data/regex.json:1370]: Remove the description. If a user want to know about the code, they can Google it.

File: pywhat/Data/regex.json [hunk 2]
```diff
@@ -1363,6 +1363,23 @@
...
+      "Rarity": 0.7,
```
Comment (piatrashkakanstantinass) [inline pywhat/Data/regex.json:1371]: The rarity seems to be too high for me

File: pywhat/Data/regex.json [hunk 3]
```diff
@@ -1363,6 +1363,23 @@
...
+      "URL": null,
+      "Tags": [
+         "Credentials"
```
Comment (piatrashkakanstantinass) [inline pywhat/Data/regex.json:1374]: Can you add `Identifiers` tag?

File: fixtures/file [hunk 4]
```diff
@@ -103,3 +103,5 @@ doi:10.1392/BC1.0
 10.1000/123
 
 a80122b2565c3e26a61cbf58d1d1aad7-us5
+
+RSSMRA00A01H501C
```
Comment (piatrashkakanstantinass) [inline fixtures/file:107]: Remove this. Fixture file is used for click tests and we do not want to slow down these.

File: tests/test_click.py [hunk 5]
```diff
@@ -737,3 +737,10 @@ def test_print_tags2():
     )
     assert result.exit_code == 0
     assert "Tags: CTF Flag" in result.output
+
+
+def test_file_fixture_italian_fiscal_code():
+    runner = CliRunner()
+    result = runner.invoke(main, ["-db", "fixtures/file"])
+    assert result.exit_code == 0
+    assert re.findall("Italian Fiscal Code", str(result.output))
```
Comment (piatrashkakanstantinass) [inline tests/test_click.py:746]: Remove this. Check my comment on fixtures file.

**Review by piatrashkakanstantinass — CHANGES_REQUESTED** [review]

**Comment by piatrashkakanstantinass** [comment]
Also, please squash the commits.

**Review by piatrashkakanstantinass — APPROVED** [review]

**Pull request merged**